lib/ui/food_detail.dart

```diff
@@ -12,53 +12,53 @@ class FoodDetail extends StatelessWidget {
-                    iconColor: colorRed,
-                  )
-                ],
-              ),
-              verticalMargin(20.0),
-              Text(
-                'Overview',
-                style: GoogleFonts.poppins(
-                    textStyle:
-                        TextStyle(fontWeight: FontWeight.w400, fontSize: 16.0)),
-              ),
-              verticalMargin(8.0),
-              Expanded(
-                child: Text(
-                  'Lorem Ipsum is simply dupesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since thes, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised the release of Letraset sheets containing Lorem Ipsum passages, and more recenp publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
-                  style: GoogleFonts.poppins(
-                      textStyle: TextStyle(color: Colors.grey, fontSize: 14.0)),
-                ),
-              )
-            ],
+            padding: EdgeInsets.all(16.0),
+            color: Colors.white,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                buildProductImage(context),
+                verticalMargin(16.0),
+                buildProductTitle(),
+                verticalMargin(20.0),
+                Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: <Widget>[
+          CustomText(
+            prefixIcon: Icons.star,
+            text: '4.5',
+            iconColor: Colors.orange,
           ),
+          CustomText(
+            prefixIcon: Icons.access_time,
+            text: '18 min',
+            iconColor: colorRed,
+          ),
+          CustomText(
+            prefixIcon: FontAwesomeIcons.locationArrow,
+            text: '2.3 km',
+            iconColor: colorRed,
+          )
+        ],
+                ),
+                verticalMargin(20.0),
+                Text(
+        'Overview',
+        style: GoogleFonts.poppins(
+            textStyle:
+                TextStyle(fontWeight: FontWeight.w400, fontSize: 16.0)),
+                ),
+                verticalMargin(8.0),
+                Expanded(
+        child: Text(
+          'Lorem Ipsum is simply dupesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since thes, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised the release of Letraset sheets containing Lorem Ipsum passages, and more recenp publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+          style: GoogleFonts.poppins(
```

Comment:
I saw that the same font is used over and again why don't you make it global from the theme in MaterialApp. @coder-Aayush

Reply:
@cimplesid ok sure. Can you check the is the card on homepage is responsive or not? @cimplesid